### PR TITLE
Update security officer

### PR DESCRIPTION
**Pull request by halkeye**
cc @wadeck

Its ironic that i'm not using the pull request template to update the pull request template.

Oh interesting, @wadeck isn't in the infra org so we can't assign him as a reviewer

### CODE REVIEW

**Review by lemeurherve — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -37,6 +37,6 @@ Make sure to implement all relevant entries (see section headers to when they ap
 - [ ] Check this if newly added person also needs to be given merge permission to the GitHub repo (please @ the people/person with their GitHub username in this issue as well). If needed, it can be done using an [IRC Bot command](https://jenkins.io/projects/infrastructure/ircbot/#github-repo-management)
 - [ ] Check that the `$pluginId Developers` team has `Admin` permissions while granting the access.
 - [ ] In the case of plugin adoption, ensure that the Jenkins Jira default assignee is either removed or changed to the new maintainer.
-- [ ] If security contacts are changed (this includes add/remove), ping the security officer (currently `@daniel-beck`) in this pull request. If an email contact is changed, wait for approval from the security officer.
+- [ ] If security contacts are changed (this includes add/remove), ping the security officer (currently `@wadeck`) in this pull request. If an email contact is changed, wait for approval from the security officer.
```
Comment (timja) [inline .github/PULL_REQUEST_TEMPLATE.md:40]:
```suggestion
- [ ] If security contacts are changed (this includes add/remove), ping the security officer (currently `@Wadeck`) in this pull request. If an email contact is changed, wait for approval from the security officer.
```

**Review by timja — APPROVED** [review]

**Comment by halkeye** [comment]
Still I think we need to get him in the org

**Comment by daniel-beck** [comment]
Already discussed a permissions review with @dduportal yesterday. Transition is still ongoing otherwise as well.

**Review by Wadeck — APPROVED** [review]
🎉

**Review by dduportal — APPROVED** [review]

**Pull request merged**